frontend/src/features/admin-form/create/featureTour/FeatureTour.tsx

```diff
@@ -9,6 +9,8 @@ interface FeatureTourProps {
   onClose: () => void
 }
 
+type JoyrideCallbackProps = Pick<CallBackProps, 'index' | 'status' | 'type'>
```

Comment:
actually rereading this. we don't control the callback props, i think its fine to not Pick the props haha. not like we can restrict the props joyride will pass to the callback

Reply:
Updated as per your comment